identifiers/country-mx/districts.csv

```diff
@@ -0,0 +1,300 @@
+ocd-division/country:mx/state:bs/fed:2,II Federal Electoral District of Baja California Sur
+ocd-division/country:mx/state:cm/fed:1,I Federal Electoral District of Campeche
+ocd-division/country:mx/state:cm/fed:2,II Federal Electoral District of Campeche
+ocd-division/country:mx/state:cs/fed:1,I Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:2,II Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:3,III Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:4,IV Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:5,V Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:6,VI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:7,VII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:8,VIII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:9,IX Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:10,X Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:11,XI Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:12,XII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:cs/fed:13,XIII Federal Electoral District of Chiapas
+ocd-division/country:mx/state:ch/fed:1,I Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:2,II Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:3,III Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:4,IV Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:5,V Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:6,VI Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:7,VII Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:8,VIII Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:ch/fed:9,IX Federal Electoral District of Chihuahua
+ocd-division/country:mx/state:co/fed:1,I Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:2,II Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:3,III Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:4,IV Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:5,V Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:6,VI Federal Electoral District of Coahuila
+ocd-division/country:mx/state:co/fed:7,VII Federal Electoral District of Coahuila
+ocd-division/country:mx/state:cl/fed:1,I Federal Electoral District of Colima
+ocd-division/country:mx/state:cl/fed:2,II Federal Electoral District of Colima
+ocd-division/country:mx/state:df/fed:1,I Federal Electoral District of Mexico City
+ocd-division/country:mx/state:df/fed:2,II Federal Electoral District of Mexico City
```

Comment:
This is listed as 'Ciudad de México', would it be better to match?

Reply:
To the extent possible that we can keep all the content in a single language, I'd like to do that. End-users who want to translate this data into other languages (E.g., Spanish) could do so without us mixing languages in the source feed.

Reply:
SG! It is reasonable to have a single language for sure. (nit: we may want to remove the accents for some of the state names, knowing that it is relatively painless for English speakers as well.)